.travis.yml

```diff
@@ -1,22 +1,18 @@
+language: ruby
+sudo: false
+
+env:
+  global:
+    - JRUBY_OPTS="-Xcli.debug=true --debug"
```

Comment:
The JRuby builds on this branch are failing hard, and I wonder if the change to JRUBY_OPTS is at fault?
```
JRUBY_OPTS="$JRUBY_OPTS --debug"
```
to
```
JRUBY_OPTS="-Xcli.debug=true --debug"
```